src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1268,6 +1300,17 @@ void runtime::cpu::CPU_ExternalFunction::build(ngraph::pass::PassConfig& pass_co
     register_common_passes(pass_manager, pass_config);
     pass_manager.run_passes(m_function, false);
 
+    static const auto do_ngraph_trace = std::getenv("NGRAPH_TRACER_LOG");
+    auto& debug_tracer = runtime::cpu::CPU_DebugTracer::getInstance();
+
+    if (do_ngraph_trace != nullptr)
+    {
+        const char* trace_log_file = std::getenv("NGRAPH_TRACER_LOG");
+        const char* bin_log_file = std::getenv("NGRAPH_BIN_TRACER_LOG");
```

Comment:
make this static. You also need to make sure this is not null or you will segfault later.